app/src/main/java/com/ranveeraggarwal/flashcards/models/FlashCard.kt

```diff
@@ -1,3 +1,13 @@
 package com.ranveeraggarwal.flashcards.models
 
-data class FlashCard (val id: Long, val question: String, val answer: String)
\ No newline at end of file
+import android.arch.persistence.room.ColumnInfo
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Index
+import android.arch.persistence.room.PrimaryKey
+import android.support.annotation.NonNull
+
+@Entity(tableName = "flashCard",
+    indices = [Index(value = ["cid"], unique = true)])
+data class FlashCard (@PrimaryKey(autoGenerate = true) @ColumnInfo(name = "cid") val id: Long,
```

Comment:
change the name to cardID it'll be neater

Reply:
and autoGenerate should be false since we will get this ID from the database itself (we can autogenerate it there)

Reply:
Since there'll only be one ID in the entire database, this works just fine. Repeating *card* everywhere is redundant.
Autogenerate point taken, will fix in the next commit.

Reply:
cid was a bit confusing when I saw it first so thought it'd be nice to have a clearer name